exes/mqtt-broadcaster/inc/config/mqtt_config.hpp

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
```

Comment:
missing include string, variant, glaze/common, cstdint

Reply:
are you referring to 

```cpp
#include <glaze/glaze.hpp>
```

Reply:
nope you do not need to include all glaze to include glz::object and glz::enumerate, it is defined in glz/core/common.hpp